src/bob/api/routes.clj

```diff
@@ -114,6 +114,12 @@
         :summary "Deletes a pipeline in a group with the specified name."
         (p/remove-pipeline group name))
 
+       (rest/GET "/pipelines" []
```

Comment:
I was planning this to have some optional query params like `?name=test&group=dev&status=passed`. These are to optionally filter the pipelines to be fetched. Are you thinking on similar lines too? Any better ideas is much appreciated too!

Reply:
Yeah, I'm implementing that. 

I'm not sure how to filter by status would be implemented, I'll find it though

Reply:
Status is defined here: https://github.com/bob-cd/bob/blob/master/resources/migrations/003-runs.edn#L19

Can use the foreign key to the pipeline.

Reply:
Thanks. Done